jsonnet/main.jsonnet

```diff
@@ -321,6 +321,7 @@ local inCluster =
     openshiftStateMetrics: openshiftStateMetrics($.values.openshiftStateMetrics),
   } +
   (import './anti-affinity.libsonnet') +
+  (import 'github.com/prometheus-operator/kube-prometheus/jsonnet/kube-prometheus/addons/ksm-lite.libsonnet') +
```

Comment:
@paulfantom please confirm it is the right place to import the addon :)

Reply:
It is :+1: 

/lgtm
/hold cancel